### ISSUE TITLE
Releasing & Distributing

### ISSUE THREAD

**Issue by jakelacey2012** [elixir-desktop/desktop#46]
Great work on creating this super easy to setup 🦾 just got a couple of questions around releasing and distributing (because I don't know much :P) - once I run `mix release` which file can I click on to run my program? It should be that file that I should be able to distribute to my users.

I think the documentation should include a section around which command to use (`mix release`) and which file to select for noobies like me.

**Comment by dominicletz** [comment]
You're right. The command to create the windows/macos/linux installer is `mix dekstop.installer`
